test/functional/services/remote/chromedriver_api/ping.js

```diff
@@ -3,7 +3,12 @@ import { fromNode as fcb } from 'bluebird';
 
 export async function ping(url) {
   try {
-    await fcb(cb => request({ url, timeout: 1000 }, cb));
+    await Promise.race([
+      fcb(cb => request(url, cb)),
+      new Promise(reject => {
```

Comment:
Doesn't the `new Promise` constructor take `resolve` as the first parameter?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise